api/routes/application/routes.py

```diff
@@ -332,3 +334,31 @@ def get_application_eoi_response(self, application):
         eoi_schema = get_round_eoi_schema(application["fund_id"], application["round_id"], application["language"])
         result = evaluate_response(eoi_schema, application["forms"])
         return result
+
+    def post_research_survey_data(self):
+        args = request.get_json()
```

Comment:
Can we have comments  ? places which required to express the task

Reply:
Added.